python/lsst/ts/LinearStage/hardware.py

```diff
@@ -123,7 +134,7 @@ def move_absolute(self, value):
 
         """
         try:
-            reply = self.send("move abs {}".format(int(value*8000)))
+            reply = self.commander.send("move abs {}".format(int(value*8000)))
```

Comment:
what's the 8000 ?

Reply:
The amount of steps approximately in one millimeter.